crates/shell/src/shells/pwsh.rs

```diff
@@ -41,14 +41,14 @@ fn join_path(value: impl AsRef<str>) -> String {
 impl Shell for Pwsh {
     fn format_env_set(&self, key: &str, value: &str) -> String {
         if value.contains('/') {
```

Comment:
I'm not sure this condition is required anymore now that we have quoting. Can probably test and remove it.

Reply:
I have tested but it can be more act as an additional check. I think we can keep it